lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -25,6 +25,9 @@
 from ops.model import Relation
 from psycopg2 import sql
 from psycopg2.sql import Composed
+from constants import DEPENDENCY_PLUGINS
```

Comment:
The lib shouldn't import from the charm. Those constants should be moved here and imported from here in the charm.